src/cp/objectives/free_days_after_night_shift_phase.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+class FreeDaysAfterNightShiftPhaseObjective:
+    KEY = "FreeDaysAfterNightShiftPhase"
+
+    def __init__(self, weight, employees, days, shifts):
+        self.weight = weight
+        self.employees = employees
+        self.days = days
+        self.shifts = shifts
+
+    def score(self, solution):
```

Comment:
I think this is called `create` with a different signature, take a look into the superclass or other objectives